survey-service/src/test/java/com/ktpt/surmoon/service/survey/integration/SurveyIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class SurveyIntegrationTest extends IntegrationTest {
+    @DisplayName("설문 생성")
+    @Test
+    void createSurvey() {
+        // given
+        Member creator = findAnyMember();
+
+        // when
+        SurveyRequest request = new SurveyRequest("title", creator.getId());
+        SurveyResponse response = post(request, SurveyController.SURVEY_URI, SurveyResponse.class);
+
+        // then
+        assertAll(
+                () -> assertThat(response.getId()).isNotNull(),
+                () -> assertThat(response.getTitle()).isEqualTo(request.getTitle()),
+                () -> assertThat(response.getCreatorId()).isEqualTo(request.getCreatorId()),
+                () -> assertThat(response.getCreatedDate()).isNotNull(),
+                () -> assertThat(response.getLastModifiedDate()).isNotNull()
+        );
+    }
+
+    @DisplayName("설문 생성 실패")
+    @Test
+    void createSurveyFails() {
+        // when
+        SurveyRequest request = new SurveyRequest("", -1L);
+        ErrorResponse response = postFails(request, SurveyController.SURVEY_URI);
+
+        // then
+        assertThat(response.getMessages())
+                .containsExactlyInAnyOrder("must not be blank", "Member with id: -1 does not exists!");
+    }
+
+    @DisplayName("설문 수정")
+    @Test
+    void updateSurvey() {
+        // given
+        Survey saved = findAnySurvey();
+        // when
+        SurveyRequest request = new SurveyRequest("changed", saved.getCreatorId());
+        SurveyResponse response = put(request, SurveyController.SURVEY_URI + "/" + saved.getId(), SurveyResponse.class);
+
+        // then
+        assertAll(
+                () -> assertThat(response.getId()).isEqualTo(saved.getId()),
+                () -> assertThat(response.getTitle()).isEqualTo(request.getTitle()),
+                () -> assertThat(response.getCreatorId()).isEqualTo(saved.getCreatorId()),
+                () -> assertThat(response.getCreatedDate()).isEqualTo(saved.getCreatedDate()),
+                () -> assertThat(response.getLastModifiedDate().isAfter(saved.getLastModifiedDate())).isTrue()
+        );
+    }
+
+    @DisplayName("설문 수정 실패")
+    @TestFactory
+    Stream<DynamicTest> updateSurveyFails() {
+        // given
+        Survey saved = findAnySurvey();
+
+        return Stream.of(
+                DynamicTest.dynamicTest("같은 title로 수정시 BadRequest", () -> {
+                    // when
+                    SurveyRequest request = new SurveyRequest(saved.getTitle(), saved.getCreatorId());
+                    ErrorResponse response = putFails(request, SurveyController.SURVEY_URI + "/" + saved.getId());
+
+                    // then
+                    assertThat(response.getMessages()).containsExactly("같은 내용으로 수정할 수 없습니다.");
```

Comment:
```suggestion
                    assertThat(response.getMessages()).containsExactly("같은 title로 수정할 수 없습니다.");
```

현재 `Survey` 내부에서 해당 검증에 대한 내용은 `title`만 확인하기에 요게 더 명확할 거 같습니다.

~~~java
        // Survey
        if (this.title.equals(title)) {
            throw new IllegalArgumentException("같은 내용으로 수정할 수 없습니다.");
        }
~~~